Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -207,6 +208,13 @@ def get_incidents(self, incident_id_list=None, lte_modification_time=None, gte_m
                 'value': gte_creation_time_milliseconds
             })
 
+        if status:
+            filters.append({
+                'field': 'status',
+                'operator': 'eq',
+                'value': status
+            })
```

Comment:
can we do single request to fetch incidents with multiple statuses ?

Reply:
No, the only operators we have are `eq` and `neq`, we do not have an `in` operator.